src/libraries/Directory.Build.targets

```diff
@@ -154,28 +154,24 @@
     <PackageReference Include="Microsoft.DotNet.GenFacades" Version="$(MicrosoftDotNetGenFacadesVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
   </ItemGroup>
 
-  <Target Name="GenerateReferenceSource">
```

Comment:
This is a breaking change which should usually be part of a batched rollout. I'm unsure if we want to wait another month for this change to go in. cc @ericstj 

I assume docs need to be updated as well?